src/frontend/src/helpers/Storage/Interface.ts

```diff
@@ -0,0 +1,80 @@
+export {AbstractStorage,
+        File, FileID, FileBrief,
+        Project, ProjectID, ProjectBrief,
+        Settings, defaultSettings}
+
+abstract class AbstractStorage {
```

Comment:
So there are times where you need both the FileID and the ProjectID -- for example, setFileToRun or whatever.  Yes, the row for the FileID would also store the ProjectID, but it's way nicer if you don't have to do a lookup.

I think in most cases you don't need the Project though, only for setFileToRun.